sktime/datatypes/_utilities.py

```diff
@@ -290,3 +290,75 @@ def get_window(obj, window_length=None, lag=0):
     raise ValueError(
         "bug in get_latest_window, unreachable condition, ifs should be exhaustive"
     )
+
+
+def get_slice(obj, start=None, end=None):
+    """Slice obj with start (inclusive) and end (exclusive) indices.
+
+    Returns time series or time series panel with time indices
+        strictly greater and equal to start index and less than
+        end index.
+
+    Parameters
+    ----------
+    obj : sktime compatible time series Series type or None
```

Comment:
can you make sure this is correct? Currently it will be incorrect if it is numpy, because no slicing is done

Reply:
erroneous comment by me, see below.